Laba2SCCS/src/BusinessLayer/Infrastructe/DateException.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Runtime.Serialization;
+
+namespace BusinessLayer.Infrastructe
```

Comment:
It would be better to have all exceptions in the Exceptions folder.
Remove this folder.